typescript/14-atbash-cipher/atbash-cipher.ts

```diff
@@ -0,0 +1,24 @@
+export function encode(plainText: string): string {
+  return translatedSymbols(plainText).replace(/(.{5})(?=.)/g, '$1 ');
+}
+
+export function decode(cipherText: string): string {
+  return translatedSymbols(cipherText);
+}
+
+function translatedSymbols(text: string): string {
+  const alphabet = "abcdefghijklmnopqrstuvwxyz";
+  const normalizedText = text.toLowerCase().replace(/[^a-z0-9]/g, '');
+  let encodedText = "";
+
+  for (let i = 0; i < normalizedText.length; i++) {
+    if (alphabet.includes(normalizedText[i])) {
```

Comment:
Can be done with a single `indexOf` call which is twice as fast as calling `indexOf + includes`